satpy/multiscene.py

```diff
@@ -99,6 +100,17 @@ def stack(datasets):
     return base
 
 
+def timeseries(datasets):
+    """Expands dataset with and concats by time dimension"""
+    expanded_ds = []
+    for ds in datasets:
+        tmp = ds.expand_dims("time")
+        tmp.coords["time"] = pd.DatetimeIndex([ds.attrs["start_time"]])
+        expanded_ds.append(tmp)
+
+    return xr.concat(expanded_ds, dim="time")
```

Comment:
Does SatPy's Scene have any issues with a DataArray with a third dimension named time? I suppose if it is only used for the geoviews time series right now it won't be a problem.

Reply:
Good question. I guess I haven't tested that in depth but I think as far as I know it was no problem. I will have a look at it.

Reply:
I think it could be a problem in `save_datasets` or resampling, but yeah this probably won't be used anywhere except for with geoviews at this point so I'm not too concerned.

Reply:
I'm not concerned about this right now. If someone wants to use time series outside of geoviews then they can implement the functionality at that time. SatPy does not have very good handling of multiple times right now and leaves it to the user to manage multiple orbits/images in most cases so this shouldn't be something you need to be concerned with for this PR.